CMake/config.cmake

```diff
@@ -134,10 +161,6 @@ write_basic_package_version_file(
 
 if (BUILD_INSTALLER)
     install(FILES "${CMAKE_CURRENT_BINARY_DIR}/CMake/rttr-config-version.cmake"
-             DESTINATION cmake
+             DESTINATION ${CMAKE_INSTALL_DATADIR}/rttr/cmake
```

Comment:
this looks now strange on windows:
![image](https://user-images.githubusercontent.com/11368160/35354238-be34cebe-0149-11e8-82d3-f92e445b7714.png)
The folder should be named cmake. I think otherwise cmake will not find the script, when pointing to the root directory

Reply:
the lib folder should be empty? isnt it?  the libraries should all go into the bin folder...?

Reply:
may be we could symlink on windows into the lib folder, and install all everythin properly in the lib folder?, why is the lib folder even constructed?

Reply:
The lib folder has to be there, there are the *.lib files placed.
necessary to link the libraries on windows

Reply:
ah, where is this path hardcoded into cmake? i didnt see that one? maybe I missed something?